css/templates/travel/page-vars.css

```diff
@@ -112,3 +137,95 @@
   @custom-media --breakpoint-md-lg (min-width: 52.06rem) and (max-width: 72rem);
   @custom-media --breakpoint-v-md-lg (min-height: 52.06rem) and (max-height: 72rem);
 }
+
+body {
+  color: var(--font-color-primary);
+  font-family: var(--font-family-primary);
+  line-height: var(--line-height-2);
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
this is probably already set in normalize or somewhere in our CSS, do not add properties again that are already inherited and have this value. 

Remember we need to fit our CSS within 50K